python/servo/bootstrap.py

```diff
@@ -391,6 +392,40 @@ def get_linux_distribution():
     return distrib, version
 
 
+TARGET_MACOS_SDK = "MacOSX10.14.sdk"
+
+
+def apple(context, force=False):
+    '''Bootstrapper for building on macOS.'''
+
+    deps_dir = os.path.join(context.sharedir, "macos-dependencies")
+    version = map(lambda x: int(x), platform.mac_ver()[0].split('.'))
+    context.macos_sdk_path = None
+
+    # Any earlier version than the 10.15 SDK is supported.
+    if version[0] == 10 and version[1] < 15:
+        return
+
+    if not os.path.isdir(deps_dir):
+        os.makedirs(deps_dir)
+
+    sdk_path = os.path.join(deps_dir, TARGET_MACOS_SDK)
+    if not os.path.isdir(sdk_path):
+        sdk_file = TARGET_MACOS_SDK + ".tar.xz"
+        archive_path = os.path.join(deps_dir, sdk_file)
+        print('Downloading SDK for macOS 10.14...')
+        sdk_url = "https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/" + sdk_file
```

Comment:
I'm not in favour of downloading and installing things off a random GH account, tbh. Why can't those be installed from Apple stuff?

Reply:
My assumption is that they're not available in a convenient place. Doesn't matter with your backport, though.